shell/platform/darwin/macos/framework/Source/FlutterBackingStoreData.mm

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterBackingStoreData.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterFrameBufferProvider.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterIOSurfaceHolder.h"
+
+#include <OpenGL/gl.h>
+
+@interface FlutterBackingStoreData () {
+  size_t _layerId;
+  FlutterFrameBufferProvider* _fbProvider;
+  FlutterIOSurfaceHolder* _ioSurfaceHolder;
```

Comment:
if you change this to property these will be synthesized.

Reply:
Done